docs/components/edge/edged.md

```diff
@@ -48,8 +48,8 @@ This module helps in monitoring pod status for edged. Every second, using probe'
 Container Runtime Interface (CRI) – a plugin interface which enables edged to use a wide variety of container runtimes, without the need to recompile and also support multiple runtimes like docker, containerd, cri-o etc
 
 #### Why CRI for edge?
-Currently kubeedge edged supports only docker runtime using the legacy dockertools.
-+ CRI support for  multiple container runtime in kubeedge is needed due to below mentioned factors
+Currently KubeEdge edged supports only the docker runtime using legacy dockertools.
++ CRI support for multiple container runtimes in KubeEdge is needed due to the factors mentioned below
```

Comment:
ditto, May need add comment to show that it has been realized.

Reply:
Agreed, the cri has been integrated at last year.

Reply:
fixed in 3db6642